Patches/OpenCV/3.1.0/common.cmake

```diff
@@ -0,0 +1,150 @@
+# This file is included from a subdirectory
```

Comment:
Again, not trying to make more work but, my preference for patching code is to do it in 2 commits. The first commit adds the file, the second commit applies the patch. That allows us to see exactly what was changed in the file.

Having said that, I think that line should be added to a developer contribution document if we have such a thing.